Client/Frontend/Strings.swift

```diff
@@ -347,6 +347,54 @@ extension String {
                 comment: "Button title which, when tapped, will allow the user to save valid credit card details.")
         }
 
+        // Remember Card
+        public struct RememberCard {
+            public static let MainTitle = MZLocalizedString(
+                key: "CreditCard.RememberCardTitle.v115",
+                tableName: "RememberCard",
+                value: "Remember this card?",
+                comment: "This value is used as the title for the remember credit card page")
+            public static let Header = MZLocalizedString(
+                key: "CreditCard.RememberCardHeader.v115",
+                tableName: "RememberCard",
+                value: "Save your card information securely with Firefox to check out faster next time.",
+                comment: "This value is used as the header for the remember card page")
+            public static let MainButtonTitle = MZLocalizedString(
+                key: "CreditCard.RememberCardPromptButtonYes.v115",
+                tableName: "RememberCard",
+                value: "Yes",
+                comment: "This value is used as the title for the Yes button in the remember credit card page")
+            public static let SecondaryButtonTitle = MZLocalizedString(
+                key: "CreditCard.RememberCardPromptButtonNotNow.v115",
+                tableName: "RememberCard",
+                value: "Not Now",
+                comment: "This value is used as the title for the Not Now button in the remember credit card page")
+        }
+
+        // Update Card
+        public struct UpdateCard {
+            public static let MainTitle = MZLocalizedString(
+                key: "CreditCard.UpdateCardPromptTitle.v115",
+                tableName: "RememberCard",
+                value: "Update this card?",
+                comment: "This value is used as the title for the update card page")
+            public static let ManageCardsButtonTitle = MZLocalizedString(
+                key: "CreditCard.UpdateCardManageCardsButtonTitle.v115",
+                tableName: "RememberCard",
+                value: "Manage cards",
+                comment: "This value is used as the title for the Manage Cards button from the update credit card page")
+            public static let YesButtonTitle = MZLocalizedString(
+                key: "CreditCard.UpdateCardPromptTitle.v115",
+                tableName: "RememberCard",
+                value: "Yes",
+                comment: "This value is used as the title for the button in the update credit card page")
+            public static let NotNowButtonTitle = MZLocalizedString(
+                key: "CreditCard.UpdateCardPromptButtonNotNow.v115",
+                tableName: "RememberCard",
+                value: "Not Now",
+                comment: "This value is used as the title for the Not Now button in the update credit card page")
```

Comment:
Fixed the tablename, and key names in there. It's easier to make sure there's no conflicts between key strings if they follow this standard.
```suggestion
            public static let MainTitle = MZLocalizedString(
                key: "CreditCard.UpdateCard.MainTitle.v115",
                tableName: "UpdateCard",
                value: "Update this card?",
                comment: "This value is used as the title for the update credit card page")
            public static let ManageCardsButtonTitle = MZLocalizedString(
                key: "CreditCard.UpdateCard.ManageCardsButtonTitle.v115",
                tableName: "UpdateCard",
                value: "Manage cards",
                comment: "This value is used as the title for the Manage Cards button from the update credit card page")
            public static let YesButtonTitle = MZLocalizedString(
                key: "CreditCard.UpdateCard.YesButtonTitle.v115",
                tableName: "UpdateCard",
                value: "Yes",
                comment: "This value is used as the title for the button in the update credit card page")
            public static let NotNowButtonTitle = MZLocalizedString(
                key: "CreditCard.UpdateCard.NotNowButtonTitle.v115",
                tableName: "UpdateCard",
                value: "Not Now",
                comment: "This value is used as the title for the Not Now button in the update credit card page")
```